projects/plugins/crm/src/automation/commons/conditions/contacts/class-contact-status-changed.php

```diff
@@ -0,0 +1,168 @@
+	 * attribute $condition_met is set to true; otherwise, it is set to false.
+	 *
+	 * @param array $data The data this condition has to evaluate.
+	 * @return void
+	 * @throws Automation_Exception If an invalid operator is encountered.
+	 */
+	public function execute( array $data ) {
+		if ( ! $this->is_valid_contact_status_changed_data( $data ) ) {
+			$this->logger->log( 'Invalid contact status changed data', $data );
+			$this->condition_met = false;
+			return;
+		}
+
+		$field    = $this->get_attributes()['field'];
+		$operator = $this->get_attributes()['operator'];
+		$value    = $this->get_attributes()['value'];
+
+		$this->logger->log( 'Condition: ' . $field . ' ' . $operator . ' ' . $value . ' => ' . $data['data'][ $field ] );
+
+		switch ( $operator ) {
+			case 'is':
+				$this->condition_met = ( $data['data'][ $field ] === $value );
+				$this->logger->log( 'Condition met?: ' . ( $this->condition_met ? 'true' : 'false' ) );
+
+				return;
+			case 'is_not':
+				$this->condition_met = ( $data['data'][ $field ] !== $value );
+				$this->logger->log( 'Condition met?: ' . ( $this->condition_met ? 'true' : 'false' ) );
+
+				return;
+		}
+		$this->condition_met = false;
+		$this->logger->log( 'Invalid operator: ' . $operator );
+
+		throw new Automation_Exception( 'Invalid operator: ' . $operator );
+	}
+
+	/**
+	 * Checks if the contact has at least the necessary keys to detect a status
+	 * change.
+	 *
+	 * @param array $contact_data The contact data.
+	 * @return bool True if the data is valid to detect a status change, false otherwise
+	 */
+	private function is_valid_contact_status_changed_data( array $contact_data ): bool {
+		return isset( $contact_data['id'] ) && isset( $contact_data['data'] ) && isset( $contact_data['data']['status'] );
+	}
+
+	/**
+	 * Get the slug for the contact status changed condition.
+	 *
+	 * @return string The slug 'contact_status_changed'.
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/condition/contact_status_changed';
+	}
+
+	/**
+	 * Get the title for the contact status changed condition.
+	 *
+	 * @return string The title 'Contact Status Changed'.
+	 */
+	public static function get_title(): string {
+		return 'Contact Status Changed';
+	}
+
+	/**
+	 * Get the description for the contact status changed condition.
+	 *
+	 * @return string The description for the condition.
+	 */
+	public static function get_description(): string {
+		return __( 'Checks if a contact status change matches an expected value', 'zero-bs-crm' );
+	}
+
+	/**
+	 * Get the type of the contact status changed condition.
+	 *
+	 * @return string The type 'condition'.
+	 */
+	public static function get_type(): string {
+		return 'condition';
+	}
+
+	/**
+	 * Get the category of the contact status changed condition.
+	 *
+	 * @return string The category 'jpcrm/contact_condition'.
+	 */
+	public static function get_category(): string {
+		return 'jpcrm/contact_condition';
```

Comment:
```suggestion
		return __( 'contact', 'zero-bs-crm' );
```

This is not like the slug. It's used to be able to search by categories. Search conditions for contact...

Reply:
[2c42777](https://github.com/Automattic/jetpack/pull/32206/commits/2c427775f047093ae571b7bcab0b1696559867ce)